python/scipp.cpp

```diff
@@ -28,7 +28,11 @@ void init_unary(py::module &);
 void init_units_neutron(py::module &);
 void init_variable(py::module &);
 
+void init_generated_comparison(py::module &);
 void init_generated_math(py::module &);
+void init_generated_trigonometry(py::module &);
+void init_generated_util(py::module &);
+void init_generated_special_values(py::module &);
```

Comment:
Should there be an umbrella `init_generated` function that the generator builds from these individual functions?

Reply:
Yes I agree, I think I will try to add something like that soon.